<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>577</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>577</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 154, 38);
background-image: url(:/resource/photo/Resource/2.jpg);

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>361</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 35pt &quot;新宋体&quot;;
</string>
    </property>
    <property name="text">
     <string>工资管理系统</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>77</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_Register">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
       </property>
       <property name="text">
        <string>注册账号</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>72</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_FindPassword">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
       </property>
       <property name="text">
        <string>找回密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>221</width>
      <height>188</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
         </property>
         <property name="text">
          <string>账号  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LineEdit_accout">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
         </property>
         <property name="text">
          <string>密码  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LineEdit_password">
         <property name="styleSheet">
          <string notr="true">border: 1px solid gray;border-radius:5px; 
padding: 0px 10px 0px 10px;
font: 12pt &quot;仿宋&quot;;
background:rgb(0, 170, 255);
selection-background-color: darkgray;
</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButton_worker">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
         </property>
         <property name="text">
          <string>员工登录</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>27</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_manager">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
         </property>
         <property name="text">
          <string>管理员登录</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_Login">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
         </property>
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
